src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchCommand.cs

```diff
@@ -66,6 +66,24 @@ public static void Register(CliCommand rootCommand, Func<ILoggerWithColor> getLo
                 Arity = ArgumentArity.ExactlyOne
             };
 
+            var format = new CliOption<string>("--format")
+            {
+                Description = Strings.pkgSearch_FormatDescription,
+                Arity = ArgumentArity.ExactlyOne
+            };
+
+            var verbosity = new CliOption<string>("--verbosity")
+            {
+                Description = Strings.pkgSearch_VerbosityDescription,
+                Arity = ArgumentArity.ExactlyOne
+            };
+
+            var configFile = new CliOption<string>("--configfile")
+            {
+                Description = Strings.pkgSearch_ConfigFileDescription,
```

Comment:
How about leveraging existing string resource that other commands use for config file option? for example

https://github.com/NuGet/NuGet.Client/blob/4ea7c85930a70c19f69293b572136e46040ff745/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L43